crates/red_knot_python_semantic/resources/mdtest/exception/control_flow.md

```diff
@@ -455,9 +455,9 @@ else:
     reveal_type(x)  # revealed: slice
 finally:
     # TODO: should be `Literal[1] | str | bytes | bool | memoryview | float | range | slice`
-    reveal_type(x)  # revealed: bool | float | slice
+    reveal_type(x)  # revealed: bool | slice | float
```

Comment:
The order here changes because I'm using `swap_remove` in `Type::display` to ensure that `Literal[True, False]` is displayed as `bool` to users. But maybe it's okay to use `remove()` in `Type::display()` rather than `swap_remove`? `remove()` is `O(n)` rather than `O(1)`, but displaying a type shouldn't really be performance-sensitive: it's usually only done as part of printing a diagnostic to the terminal